packages/iov-tendermint-rpc/src/rpcclient.ts

```diff
@@ -22,6 +22,10 @@ export interface RpcStreamingClient extends RpcClient {
   readonly listen: (request: JsonRpcRequest) => Stream<JsonRpcEvent>;
 }
 
+export function instanceOfRpcStreamingClient(client: RpcClient): client is RpcStreamingClient {
+  return typeof (client as any).listen === "function";
```

Comment:
Nice check... I should fix some of my guards to be like this. (I think I check for existence of the key, not even the type of the value)